modules/ROOT/pages/license-key.adoc

```diff
@@ -0,0 +1,83 @@
+= License key
+:description_short: {productname} License Configuration Guide | {productname}
+:description: Learn how to configure license key and stop {productname} from running in the evaluation mode.
```

Comment:
@ShiridiGandham suggested copy edit.
```suggestion
:description: Learn how to configure a {productname} license key to prevent {productname} from operating in evaluation mode.
```